src/wallet/mod.rs

```diff
@@ -2364,7 +2385,22 @@ pub(crate) mod test {
 
     #[test]
     fn test_create_tx_default_sequence() {
-        let (wallet, _, _) = get_funded_wallet(get_test_wpkh());
+        let descriptors = testutils!(@descriptors (get_test_wpkh()));
+        let wallet = Wallet::new(
+            &descriptors.0,
+            None,
+            Network::Regtest,
+            AnyDatabase::Memory(MemoryDatabase::new()),
+        )
+        .unwrap();
+
+        let tx_meta = testutils! {
+                @tx ( (@external descriptors, 0) => 50_000 ) (@confirmations 1)
+        };
+
+        // Add the transaction to our db, but do not sync the db.
```

Comment:
Can we add a comment here explaining why we don't want to sync the db?

Reply:
Done.. Yes would be helpful..